content/en/blog/_posts/seven-kubernetes-pitfalls-and-how-to-avoid.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: blog
+title: "7 Common Kubernetes Pitfalls (and How I Learned to Avoid Them)"
+date: 2024-12-27

```

Comment:
This change will help us accept the article separately from deciding when it can publish:
```suggestion
date: 2024-12-27
draft: true
```